test/query.js

```diff
@@ -84,17 +84,17 @@ test('query random dataset', function (t) {
     })
   db.batch(batch, function (err) {
     t.error(err)
-
-    db.query(bbox, function (err, elements) {
-      t.error(err)
-      t.ok(Array.isArray(elements))
-      t.equals(elements.length, 100)
-    })
-
-    collect(db.query(bbox), function (err, elements) {
-      t.error(err)
-      t.ok(Array.isArray(elements))
-      t.equals(elements.length, 100)
+    db.ready(function () {
```

Comment:
woah! if this was necessary to get the test passing, then something is very wrong, since `db.query` already waits on indexes to be ready.

update: tested /wo `ready` and it still passes, woo